prusa/connect/printer/__init__.py

```diff
@@ -393,6 +393,32 @@ def get_info(self) -> Dict[str, Any]:
                     sheet_settings=self.sheet_settings,
                     active_sheet=self.active_sheet)
 
+    def get_info_legacy(self) -> Dict[str, Any]:
```

Comment:
are we really need that?

Reply:
I suppose not, good point, I'll remove it and I'll change the call in the PrusaLink